### PR TITLE
Remove HammerJS and lodash dependencies

### PR DESCRIPTION
**Pull request by evanuk**
Uses $swipe instead of HammerJS.  Needed to turn Card and Stack into Angular services in order to accomplish this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
     "url": "http://gajus.com"
   },
   "dependencies": {
-    "hammerjs": "^2.0.4",
-    "lodash": "^4.6.1",
+    "angular": "^1.5.4",
+    "angular-touch": "^1.6.1",
```
Comment (dpogue) [inline package.json]: Should probably not mix-n-match these, even if the `^` means they'll both end up being 1.6.x

File: src/utilities.js [hunk 2]
```diff
@@ -8,17 +7,15 @@ import _ from 'lodash';
  * @returns {Array}
  */
 const elementChildren = (element) => {
-  return _.filter(element.childNodes, {
-    nodeType: 1
-  });
+  return Array.from(element.childNodes).filter(elem => elem.nodeType === 1);
```
Comment (dpogue) [inline src/utilities.js]: Array.from isn't supported by IE11.

`element.childNodes` should be a DOMCollection, so I *think* you can do something like this
```js
Array.prototype.filter.call(element.childNodes, elem => elem.nodeType === 1)
```

File: src/Card.js [hunk 3]
```diff
@@ -411,7 +405,7 @@ Card.makeConfig = (config = {}) => {
     transform: Card.transform
   };
 
-  return _.assign({}, defaultConfig, config);
+  return Object.assign({}, defaultConfig, config);
```
Comment (isuda) [inline src/Card.js]: Object.assign won't work in IE11.

Comment (dpogue) [inline src/Card.js]: Might need a polyfill for this (I had one inline in AyUser where I needed it: https://github.com/AyogoHealth/ayogo-client-common/blob/master/src/user/user.ts#L50)

File: src/utilities.js (hunk 2 above)
Comment (isuda) [inline src/utilities.js]: Array.from does not work in IE11

**Review by isuda — CHANGES_REQUESTED** [review]

**Comment by evanuk** [comment]
If its building to a dist folder it should be okay, right? (babel)

File: src/Stack.js [hunk 4]
```diff
@@ -99,10 +99,7 @@ angular.module(modName, ['card'])
      * @returns {Card|null}
      */
     stack.getCard = (element) => {
-      const group = _.find(index, {
-        element
-      });
-
+      const group = index[index.findIndex(i => element.id === i.element.id)];
```
Comment (dpogue) [inline src/Stack.js]: `findIndex` doesn't work in IE11 either :(

**Review by dpogue — APPROVED** [review]

**Pull request merged**